environment/alarms.tf

```diff
@@ -48,6 +48,32 @@ resource "aws_cloudwatch_metric_alarm" "nginx_errors" {
   alarm_actions       = [data.aws_sns_topic.alerts.arn]
 }
 
+resource "aws_cloudwatch_log_metric_filter" "queued_documents" {
+  name           = "MonitorQueuedDocuments.${local.environment}"
+  pattern        = "{ $.eventType = \"Queued_Documents\" }"
+  log_group_name = aws_cloudwatch_log_group.monitoring_lambda.name
+
+  metric_transformation {
+    name      = "QueuedGreaterThanHour.${local.environment}"
+    namespace = "Test"
```

Comment:
Change this from "Test" or is it a genuine name?

Reply:
No! Good call!